sui_programmability/examples/games/sources/drand_based_scratch_card.move

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// A basic game that depends on randomness from drand (chained mode).
+///
+/// Anyone can create a new game by depositing X*100 SUIs as a reward, and setting the current drand round as the base
```

Comment:
**Let's make clear that the prize is fixed and independent on the num purchased tickets**. This is an important detail and the most common question (also the pattern on when this single write approach can be applied securely).